src/html5sortable.ts

```diff
@@ -61,12 +61,12 @@ const _removeItemEvents = function (items) {
   _off(items, 'mouseleave')
 }
 
-// remove container eventes 
-const _removeContainerEvents = () => {
-  if(originContainer) {
+// Remove container events
+const _removeContainerEvents = function () {
```

Comment:
Why did you change this to use the `function` keyword?

Reply:
I changed it to be the same as the function above, maybe we could change all functions to arrow functions...